.cloudbuild/scripts/cmd/unit-tests/main.go

```diff
@@ -179,8 +179,9 @@ func run() error {
 func runUnitTests(workspace string) error {
 	cmd := exec.Command("make", "test")
 	cmd.Dir = workspace
-	cmd.Env = append(os.Environ(), "TELEPORT_ETCD_TEST=yes")
-	cmd.Env = append(os.Environ(), "TELEPORT_XAUTH_TEST=yes")
+	cmd.Env = os.Environ()
+	cmd.Env = append(cmd.Env, "TELEPORT_ETCD_TEST=yes")
+	cmd.Env = append(cmd.Env, "TELEPORT_XAUTH_TEST=yes")
```

Comment:
```suggestion
	cmd.Env = append(os.Environ(), "TELEPORT_ETCD_TEST=yes", "TELEPORT_XAUTH_TEST=yes")
```

less lines = less bugs, literally in this case. :)

Reply:
I was thinking about that, but at least in a few places saw the appends split into individual lines. I think that is more `git-blame` friendly? Not sure if there are other rationales for one or the other style.